apps/guardian-ui/public/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "fm_config_api": "ws://127.0.0.1:18138"
+}
```

Comment:
Rather than hard-coding this, let's have webpack provide it based on the env vars set. Otherwise this will be kind of difficult to modify after build.

Reply:
Isn't the idea to be able to modify after build?

Reply:
I think we should hash out how this should work for all use-cases, since they're all quite different:

* Development
  * Typically env var based, not file editing based
  * Should have a default for people running manually, and for people using mprocs / devimint
  * Should _ideally_ also fetch `config.json` so that the development experience isn't drastically different from production
* Docker
  * Should allow for shelling in and modifying `config.json`
  * Should allow for extending the image and modifying the `config.json`
  * Should allow for providing env vars _at run time_ (not image build time)
* Static tarball
  * Should only allow for modifying `config.json`
  * Should **not** use environment variables from build time

With all these considered, I think we should never use `process.env.VAR` in typescript. Otherwise, builds will have those environment variables hard coded in. All code paths should fetch the `config.json`, the only difference should be how that file is populated.

I think for development, we'll either need to extend the webpack dev server (See https://create-react-app.dev/docs/proxying-api-requests-in-development/). For docker, we'll want to update the `config.json` when the container is run based on env vars. For the tarball, we should probably provide a default `config.json` but expect the user to update it with their own values.